hushline/routes/profile.py

```diff
@@ -119,12 +119,21 @@ def submit_message(username: str) -> Response | str:
             db.session.commit()
 
             if uname.user.enable_email_notifications:
-                if not uname.user.email_include_message_content:
-                    email_body = "You have a new Hush Line message! Please log in to read it."
+                if uname.user.email_include_message_content:
```

Comment:
We don't have a test for this fairly critical path. Also might want to pessimistically check the body is encrypted on the backend and reencrypt.

Reply:
Re-encrypting on the backend won't work if the form has any encrypted fields, because then they'll be double-encrypted, and so won't be readable in the email client. Adding a test is a good idea though

Reply:
Maybe at least test that the PGP start string exists at all in the message and then encrypt. Maybe that's annoying UX in some edge cases, but it would otherwise leak data which is worse imo.

Reply:
Instead of re-encrypting, I've changed it so that if `encrypted_email_body` is not PGP-encrypted, it falls back to the generic "You have a new Hush Line message" email

Reply:
And I wrote notifications tests!